src/django_peerctl/models/peerctl.py

```diff
@@ -2491,6 +2507,8 @@ def get_data(self):
             member = [member]
 
         data.update(**device.peer_groups_netom0_data(net, members=member))
+        if len(data["peer_groups"]) == 0:
+            data["peer_groups"] = DUMMY_PEER_DATA
```

Comment:
lets replace `local_as` with net.asn @jason20c

Reply:
fixed and changed DUMMY_PEER_DATA into a function